ambillion-ui/src/components/products/productsList.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import { Icon } from '@iconify/react';
+import { TableFilter } from 'components/common/table/tableFilter';
+import DataTable from 'react-data-table-component';
+import { productTableColumns, ActionsColumn, ProductDataRow } from 'utils/table/columns';
+import { dummyProductTableData } from 'utils/table/data';
+import { ConfirmationModal } from 'components/common/modal/confirmationModal';
+import { AddProductModal } from 'components/common/modal/addProductModal';
+
+const customStyles = {
+    rows: {
+        style: {
+            minHeight: '50px' // override the row height
+        }
+    },
+    headCells: {
+        style: {
+            paddingLeft: '6px', // override the cell padding for head cells
+            paddingRight: '6px',
+            fontSize: '14px',
+            fontWeight: 'bold'
+        }
+    },
+    cells: {
+        style: {
+            paddingLeft: '6px', // override the cell padding for data cells
+            paddingRight: '6px'
+        }
+    }
+};
+
+export const ProductList = () => {
+    const [filterText, setFilterText] = useState('');
+    const [resetPaginationToggle, setResetPaginationToggle] = useState<boolean>(false);
+    const [productData, setProductData] = useState(dummyProductTableData);
+    const [isAddProductModalOpen, setIsAddProductModalOpen] = useState<boolean>(false);
+    const [showConfirmationModal, setShowConfirmationModal] = useState<boolean>(false);
+    const [deleteItemId, setDeleteItemId] = useState<number | null>(null);
+
+    const filteredItems = productData.filter((item) =>
+        item.productDisplayName?.toLowerCase().includes(filterText.toLowerCase())
+    );
+
+    //Product Add Logic
+    const handleOpenAddProductModal = () => {
+        setIsAddProductModalOpen(true);
+    };
+
+    //Product Delete Logic
+    const handleDelete = (id: number) => {
```

Comment:
remove Check if same Custom styles is used in users table if yes then make it reusable. instend pass id to the modal and from there pass it as a callback.